conda/recipes/libcudf/meta.yaml

```diff
@@ -114,6 +114,7 @@ test:
     - test -f $PREFIX/include/cudf/io/detail/json.hpp
     - test -f $PREFIX/include/cudf/io/detail/orc.hpp
     - test -f $PREFIX/include/cudf/io/detail/parquet.hpp
+    - test -f $PREFIX/include/cudf/io/detail/utils.hpp
```

Comment:
Is this a new file that was inadvertently missed?  I'm trying to compile the branch in preparation for the required JNI changes, and it doesn't build because the compiler cannot find this file.

Reply:
Yep, I had missed that file.